internal/pkg/service/stream/keboolasink/bridge/bridge.go

```diff
@@ -20,6 +20,9 @@ const (
 	stagingFileProvider = stagingModel.FileProvider("keboola")
 	// targetProvider marks files which destionation is a Keboola table.
 	targetProvider = targetModel.Provider("keboola")
+
+	// exportMetaKey is a key of the table metadata that marks each table created by the stream.export.
+	exportMetaKey = "KBC.stream.export.id"
```

Comment:
We had discussion on that. If am I right this is bridge (mapping) between new API where we have `source` and `sink` and old API where we have `receiver` and `export`. If the bridge is really bridge than this is correct. If not it can be renamed. I would leave it as topic when Michal will come.

Reply:
update: [slack](https://keboolaglobal.slack.com/archives/C054DF6DZ1B/p1719557494236309?thread_ts=1689932533.402409&cid=C054DF6DZ1B)